cmd/install/addon/cmd.go

```diff
@@ -197,72 +196,53 @@ func run(cmd *cobra.Command, argv []string) {
 		parameters.Each(func(param *cmv1.AddOnParameter) bool {
 			var val string
 			var hasVal bool
+			var optionStrings []string
+
+			options, ok := param.GetOptions()
+			if ok {
+				for _, opt := range options {
+					optionStrings = append(optionStrings, opt.Value())
+				}
+			}
```

Comment:
NP: I think we can safely get rid of the ok check here
```suggestion
			for _, opt := range param.Options() {
				optionStrings = append(optionStrings, opt.Value())
			}
```